configure.ac

```diff
@@ -271,11 +271,11 @@ if test "x$enable_debug" = xyes; then
   if test "x$CXXFLAGS_overridden" = xno; then
 	CXXFLAGS=""
   fi
-  # Prefer -Og, fall back to -O0 if that is unavailable.
+  # Prefer -O0, fall back to -Og if that is unavailable.
```

Comment:
I think it's fine to add `-O0` unconditionally. E.g.: I would be very surprised if a compiler supports `-Og` but not `-O0`,